sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -136,6 +139,7 @@
         }
         versionCreated = getVersionCreated(mappingMap);
         versionUpgraded = getVersionUpgraded(mappingMap);
+        closed = getClosed(mappingMap);
```

Comment:
Yes, I think so - It's also a flag on our side. In most cases they will be mirrored but, for example, closing a partitioned table that has no records in it would fail if we used the CLOSED state on the index, since the table would not have any indexes at that point, I think? I may be incorrect about this.